### PR TITLE
Clarify no proxy option

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description

https://jira.mesosphere.com/browse/DOCS-1960

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by joel-hamill** [comment]
@jsancio @jenafermiller  @surdy how does this look?

File: 1.9/installing/custom/configuration/examples.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+n---
```
Comment (jsancio) [inline 1.9/installing/custom/configuration/examples.md]: Is this intended?

**Review by jsancio — APPROVED** [review]
One small comment. LGTM. :sheep: it!

File: 1.10/installing/custom/configuration/configuration-parameters.md [hunk 2]
```diff
@@ -435,7 +435,9 @@ Indicates whether to enable the DC/OS proxy.
     **Important:** The specified proxies must be resolvable from the provided list of [resolvers](/docs/1.10/installing/custom/configuration/examples/#resolvers).
     *  `http_proxy: http://<user>:<pass>@<proxy_host>:<http_proxy_port>` The HTTP proxy.
     *  `https_proxy: https://<user>:<pass>@<proxy_host>:<https_proxy_port>` The HTTPS proxy.
-    *  `no_proxy: - .<(sub)domain>` A YAML nested list (-) of addresses to exclude from the proxy.
+    *  `no_proxy`: A YAML nested list (`-`) of subdomains to exclude from forwarding to the `https_proxy`. If the address matches one of these strings, or the host is within the domain of one of these strings, transactions with that node are not proxied. For example, the `no_proxy` list can be a list of internal IP addresses and Mesos DNS entries.
```
Comment (surdy) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: I'm not sure what we mean by "Mesos DNS entries"

File: 1.9/installing/custom/configuration/examples.md (hunk 1 above)
Comment (surdy) [inline 1.9/installing/custom/configuration/examples.md]: Yup was this intentional ?

Comment (joel-hamill) [inline 1.9/installing/custom/configuration/examples.md]: nope

File: 1.10/installing/custom/configuration/configuration-parameters.md (hunk 2 above)
Comment (joel-hamill) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: @jsancio ^^

Comment (jsancio) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: @surdy How do you suggest we change this?

Comment (surdy) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: I'm not sure what that stands for ? What is a mesos dns entry ? I'm completely blanking on it

Comment (jsancio) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: https://github.com/mesosphere/mesos-dns?

Comment (surdy) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: @jsancio I'm familiar with MesosDNS. Still not sure what is "Mesos DNS entries" that we would need to specify for no_proxy

Comment (surdy) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: @jsancio did you mean `.mesos` TLD ?

Comment (jsancio) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Yes. This is a better wording. How about "For example, the `no_proxy` list can be a list of internal IP addresses and `.mesos` (the TLD for Mesos DNS)"?

Comment (surdy) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: `.mesos` is already added automatically to no_proxy by the DC/OS installer https://github.com/dcos/dcos/blob/a361bbceeb14ab313c83f59c1bcfca23d6110bc3/gen/calc.py#L380. So the user is not required to add that.

Comment (surdy) [inline 1.10/installing/custom/configuration/configuration-parameters.md]: Also I am wondering what would be a better phrase for `transactions with that node are not proxied` . Would `http(s) requests to that node are no proxied` (probably can be improved further). Just want to make sure we are being clear.

**Comment by joel-hamill** [comment]
@jsancio @surdy let me know what you think https://github.com/dcos/dcos-docs/pull/1130/commits/b3781386bc4648523b0f7d0f16edf1fa06802448

**Review by surdy — APPROVED** [review]

**Review by jsancio — APPROVED** [review]
LGTM. :fireworks:

**Pull request merged**